build.gradle

```diff
@@ -739,3 +741,10 @@ task pusher(type: KotlinBuildPusher){
     overrideConfigurationId = System.getenv("TEAMCITY_OVERRIDE_NATIVE_ID") ?: "Kotlin_KotlinDev_DeployMavenArtifacts_OverrideNative"
     token = project.findProperty("teamcityBearToken") ?: System.getenv("TEAMCITY_BEAR_TOKEN")
 }
+
+CompilationDatabaseKt.mergeCompilationDatabases(project, "compdb", [
```

Comment:
It might be useful to make this task a depedency of some frequently executed task, so that compilation database would have a chance to regenerate on a regular basis. The job itself is rather inexpensive compared to compilation, so having it as a dependency won't have a noticable impact.